sys/xtimer/xtimer_core.c

```diff
@@ -154,7 +153,8 @@ static bool _timer_comparison(xtimer_t* timerA, xtimer_t* timerB)
         return true;
     }
     if (timerA->long_offset == timerB->long_offset
-        && timerA->start_time + timerA->offset - timerB->start_time <= timerB->offset) {
+        && (timerA->start_time + timerA->offset < timerB->start_time
+            || timerA->start_time + timerA->offset - timerB->start_time <= timerB->offset)) {
```

Comment:
the <= conflicts with the doc saying "earlier than"

Reply:
Will fix the doc above.